src/CsharpClient/Quix.Streams.Streaming/QuixStreamingClient.cs

```diff
@@ -51,7 +51,7 @@ public class QuixStreamingClient
         /// <summary>
         /// The base API uri. Defaults to <c>https://portal-api.platform.quix.ai</c>, or environment variable <c>Quix__Portal__Api</c> if available.
         /// </summary>
-        public Uri ApiUrl = new Uri("https://portal-api.platform.quix.io");
+        public Uri ApiUrl = new Uri("https://portal-api.platform.quix.ai");
```

Comment:
we should remember to change this when the platform domain change is done. Seems like the thing we would forget :S

Reply:
platform already injects it, only going to be a problem locally, but there will be an overlap where both will work.